applications/tari_base_node/src/commands/cli.rs

```diff
@@ -105,14 +108,21 @@ fn multiline_find_display_length(lines: &str) -> usize {
 
 /// Try to resize terminal to make sure the width is enough.
 /// In case of error, just simply print out the error.
-fn resize_terminal_to_fit_the_box(width: usize, height: usize) {
-    if let Err(e) = execute!(stdout(), SetSize(width as u16, height as u16)) {
-        println!("Can't resize terminal to fit the box. Error: {}", e)
+fn apply_terminal_properties(width: usize, height: usize, resize_terminal: bool) {
```

Comment:
Rather rename this function to what it does, remove the `if` and allow the caller to determine if they want to call it outside of the function. 

```suggestion
fn resize_terminal(width: usize, height: usize) {
```